test/component/uk/gov/hmrc/individualsemploymentsapi/controller/v2/LiveEmploymentsControllerSpec.scala

```diff
@@ -440,6 +465,8 @@ class LiveEmploymentsControllerSpec extends BaseSpec {
 
     scenario(s"IF returns an Bad Request Error") {
 
+      val employerRef = "247%2FZT6767895A"
```

Comment:
and here... May need to change this if the IF schema does not allow the `%2F`. Have we had a look at the latest schema?